.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
   "tabWidth": 4,
+  "printWidth": 100,
```

Comment:
changed the max line length from the default of 80

Reply:
140 is the best number.